python/test/test_taskqueue.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+import pytest
+
+from neuroglancer.pipeline.tasks import RegisteredTask
+from neuroglancer.pipeline import TaskQueue
+
+TRAVIS_BRANCH = None if 'TRAVIS_BRANCH' not in os.environ else os.environ['TRAVIS_BRANCH']
+
+if TRAVIS_BRANCH is None:
+  QUEUE_NAME = 'test-pull-queue'
+elif TRAVIS_BRANCH == 'master':
+  QUEUE_NAME = 'travis-pull-queue-1'
+else:
+  QUEUE_NAME = 'travis-pull-queue-2'
+
+class MockTask(RegisteredTask):
+  def __init__(self):
+    super(MockTask, self).__init__()
+
+def test_single_threaded_insertion():
+  global QUEUE_NAME
+  tq = TaskQueue(n_threads=0, queue_name=QUEUE_NAME)
+
+  n_inserts = 5
+
+  tq.purge()
+
+  try:
+    for _ in xrange(n_inserts):
+      task = MockTask()
+      tq.insert(task)
+
+    lst = tq.list()
```

Comment:
I sometimes also use blank lines to indicate something significant or time consuming happened.